client/src/pages/ManageBookings/ManageBookings.tsx

```diff
@@ -0,0 +1,199 @@
+import { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import PageContainer from '../../components/PageContainer/PageContainer';
+import { Box, Grid, Typography, Avatar, CircularProgress } from '@mui/material';
+import useStyles from './useStyles';
+import CardWrapper from '../../components/CardWrapper/CardWrapper';
+import getSitterRequests from '../../helpers/APICalls/sitterRequests';
+import changeStatus from '../../helpers/APICalls/changeStatus';
+import { useAuth } from '../../context/useAuthContext';
+import MyCalendar from '../../components/MyCalendar/MyCalendar';
+import { format } from 'date-fns';
+import RequestInfo from './RequestInfo/RequestInfo';
+import { RequestApiData } from '../../interface/RequestApiData';
+
+export default function ManageBookings(): JSX.Element {
+  const classes = useStyles();
+  const history = useHistory();
+  const { loggedInUser } = useAuth();
+  const { sitterRequests } = useAuth();
+  const { updateSitterRequestsContext } = useAuth();
+
+  const handleStatus = (
+    accepted: boolean,
+    declined: boolean,
+    requestId: string,
+    sitterId: string,
+    avatar?: string | undefined,
+  ): void => {
+    changeStatus(accepted, declined, requestId, sitterId, avatar).then((res) => {
+      const index = sitterRequests.findIndex((request: RequestApiData) => {
+        return request._id === requestId;
+      });
+      if (index !== -1) {
+        const newSitterRequests = [...sitterRequests];
+        newSitterRequests[index] = res;
+        console.log(res);
```

Comment:
Please watch for unnecessary logs to the console. Going forward, we should be cleaning up our logs done during development. Thanks!